### PR TITLE
change "w/" to samples with

### PR DESCRIPTION
**Pull request by dippindots**
Fix https://github.com/cBioPortal/cbioportal/issues/5770.

### CODE REVIEW

**Comment by dippindots** [comment]
Do we need to distinguish between singular and plural for the sample size?

**Comment by alisman** [comment]
@dippindots do you mean with "1 with datum"  ? vs "2 with data" ?

**Comment by dippindots** [comment]
@alisman I mean like "0 sample" "1 sample" "2 samples", Or we always use samples for all of them?

**Comment by dippindots** [comment]
@alisman Thanks for your answer, update soon.

File: src/pages/studyView/tabs/SummaryTab.tsx [hunk 1]
```diff
@@ -335,7 +335,7 @@ export class StudySummaryTab extends React.Component<IStudySummaryTabProps, {}>
                                         onChange={this.props.store.toggleWithMutationDataFilter}
                                         disabled={this.props.store.molecularProfileSampleCounts.result.numberOfMutationProfiledSamples === undefined}
                                     >
-                                        {this.props.store.molecularProfileSampleCounts.result.numberOfMutationProfiledSamples === undefined ? '0' : this.props.store.molecularProfileSampleCounts.result.numberOfMutationProfiledSamples.toLocaleString()} w/ mutation data
+                                        {this.props.store.molecularProfileSampleCounts.result.numberOfMutationProfiledSamples === undefined ? '0 sample' : this.props.store.molecularProfileSampleCounts.result.numberOfMutationProfiledSamples.toLocaleString() + (this.props.store.molecularProfileSampleCounts.result.numberOfMutationProfiledSamples <= 1 ? " sample" : " samples")} with mutation data
```
Comment (adamabeshouse) [inline src/pages/studyView/tabs/SummaryTab.tsx]: this line is so long, can we save `const numMutationProfiledSamples = this.props.....numberOfMutationProfiledSamples` and reuse it

File: src/pages/studyView/tabs/SummaryTab.tsx [hunk 2]
```diff
@@ -350,7 +350,7 @@ export class StudySummaryTab extends React.Component<IStudySummaryTabProps, {}>
                                         onChange={this.props.store.toggleWithCNADataFilter}
                                         disabled={this.props.store.molecularProfileSampleCounts.result.numberOfCNAProfiledSamples === undefined}
                                     >
-                                        {this.props.store.molecularProfileSampleCounts.result.numberOfCNAProfiledSamples === undefined ? '0' : this.props.store.molecularProfileSampleCounts.result.numberOfCNAProfiledSamples.toLocaleString()} w/ CNA data
+                                        {this.props.store.molecularProfileSampleCounts.result.numberOfMutationProfiledSamples === undefined ? '0 sample' : this.props.store.molecularProfileSampleCounts.result.numberOfMutationProfiledSamples.toLocaleString() + (this.props.store.molecularProfileSampleCounts.result.numberOfMutationProfiledSamples <= 1 ? " sample" : " samples")} with CNA data
```
Comment (adamabeshouse) [inline src/pages/studyView/tabs/SummaryTab.tsx]: both here and above, the english idiom is "0 samples, 1 sample, 2(+) samples". doesnt make much sense but thats how it is :D

Comment (zhx828) [reply src/pages/studyView/tabs/SummaryTab.tsx]: I actually just knew. In this case, there are quite few places in the study view need to be updated.

**Pull request merged**